Resources/Prototypes/Magic/forcewall_spells.yml

```diff
@@ -13,3 +13,18 @@
   serverEvent: !type:InstantSpawnSpellEvent
     prototype: WallForce
     posData: !type:TargetInFront
+
+- type: instantAction
```

Comment:
does the mime not already have a power that's just making an invisible force wall?

Reply:
they do but it's not 3 wide iirc

Reply:
Normal mime wall is one tile infront of them, the mimebook mime wall is 3 tiles wide instead

Reply:
I think it's better to have one or the other, not both.